src/tests/intellij/test_advanced_options.py

```diff
@@ -47,6 +49,7 @@ def test_skip_reports_intellij(configurations, setup_intellij, app_name, analysi
 
 @pytest.mark.parametrize("app_name", ["weblogic_to_eap7_legacy_reports"])
 @pytest.mark.parametrize("ide", ["intellij"])
+@pytest.mark.intellij
```

Comment:
You could apply markers at the module level. Maybe, there is a way to apply it at the directory level as well.
[Markers at module level ]((https://docs.pytest.org/en/7.1.x/reference/reference.html#globalvar-pytestmark)

Reply:
@nachandr  Thanks for the feedback, according to the documenation applying the markes at module level requires to implement a class, when I did that it showed unrecognized variables errors that required changing hwo the tests are structured to fix those variable errors, I think it's best to just add it to each test it's not much work